common/lib/xmodule/xmodule/course_module.py

```diff
@@ -386,6 +386,18 @@ class CourseFields(object):
         ),
         scope=Scope.settings
     )
+    enable_ccx = Boolean(
+        # Translators: Custom Courses for edX (CCX) is an edX feature for re-using course content. CCX Coach is
+        # a role created by a course Instructor to enable a person (the "Coach") to manage the custom course for
+        # his students.
+        display_name=_("Enable CCX"),
+        help=_(
```

Comment:
Sorry @sarina, done with fix. By the way I get confuse on this comment ( https://github.com/edx/edx-platform/pull/8207#discussion_r31268608) where you said **Also the second sentence is probably not necessary. It is probably sufficient to say:** I thought you want me to delete second comment.
 
Thanks for explanation.